### ISSUE TITLE
Syncthing indicator icon disappeared

### ISSUE THREAD

**Issue by hackel** [syncthing/syncthing-gtk#454]
I'm not sure what has changed, but a I can no longer see the indicator for syncthing-gtk.  I'm running Ubuntu 18.04 with gnome-shell 3.28.1 and the Ubuntu appindicators/kstatusnotifieritem extension.  syncthing-gtk 0.9.3.1 was installed from the Opensuse deb repo.

This means that initially, syncthing-gtk shows no UI at all, but it is running (I have autostart enabled).  If I launch it again, the window appears and it works fine.

I noticed this in the debug output that doesn't look right:
```
I StatusIcon    Using backend StatusIconGTK3 (primary)
I App           Syncthing-GTK started and running in notification area
W StatusIcon    StatusIcon: Using backend StatusIconDummy (fallback)
```

**Comment by kozec** [comment]
> ```
> I StatusIcon    Using backend StatusIconGTK3 (primary)
> W StatusIcon: Using backend StatusIconDummy
> ```

Sounds like GTK reporting there is no tray. On other hand, on Ubuntu, AppIndicator should be used.

Please, try starting ST-GTK from terminal with SYNCTHING_STATUS_BACKEND set like this:

`SYNCTHING_STATUS_BACKEND=AppIndicator syncthing-gtk`

**Comment by hackel** [comment]
Thanks, I just tried that after finding it in another issue, and it just lead to "StatusIcon: Requested backend StatusIconAppIndicator is not supported".  That lead me to double-check the dependencies.

The package `gir1.2-appindicator3-0.1` got removed during an upgrade.  Installing it fixed it.  You might consider adding it to the "Recommends" section of the Debian package.  Thank you for the quick response!
